### PR TITLE
basic introduction link leads to 404 on Readme

### PR DESCRIPTION
**Pull request by nterray**
basic-introduction.md file is located in docs subfolder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ $ composer require rubix/ml
 Read the latest documentation [here](https://docs.rubixml.com/en/latest/).
 
 ## Getting Started
-If you are new to machine learning, we recommend the [basic introduction](basic-introduction.md) to get started. From there, you can browse the [example projects](https://github.com/RubixML) and tutorials listed below which range from beginner to advanced skill level.
+If you are new to machine learning, we recommend the [basic introduction](docs/basic-introduction.md) to get started. From there, you can browse the [example projects](https://github.com/RubixML) and tutorials listed below which range from beginner to advanced skill level.
```
Comment (andrewdalpino) [inline README.md]: Thank you for catching this @nterray and thank you for the PR

We actually intended to have this link point to https://docs.rubixml.com/en/latest/basic-introduction.html

Would you be able to make that change for us real quick?

**Review by andrewdalpino — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nterray) [reply README.md]: Sure :)

**Review by nterray — COMMENTED** [review]
Oops, sorry for the typo in the commit message.

**Review by andrewdalpino — APPROVED** [review]
Thank you again!

**Comment by nterray** [comment]
Thanks for the merge.

I also think that the block "Text Sentiment Analysis From Movie Reviews" on https://rubixml.com/ should point to https://github.com/RubixML/Sentiment instead of Credit. However I don't know where are the sources of the website, therefore I will let you fix it by yourself ;)

**Pull request merged**